index.js

```diff
@@ -13,9 +13,9 @@ try {
  * Defer.
  */
 
-var defer = typeof process === 'undefined' || typeof process !== 'function'
-  ? function(fn){ setTimeout(fn); }
-  : process.nextTick;
+var defer = typeof process !== 'undefined' && typeof process.nextTick === 'function'
```

Comment:
Hi @stephenmathieson I'll have to thing about it and get back at some point. Basically I'm just concerned that it is not a breaking change. Today `process = null` works fine. Just make sure that keeps working however you think is good.

> if it's null, you're right, there's an issue here, but i can't think of any practical case where process === null.

I already gave you an example: In a web browser, since process has no meaning, it can be any value if someone is using that for something. That's the whole purpose of this line.